io/driver/module_unload_load.sh

```diff
@@ -13,13 +13,15 @@
 #
 # Copyright: 2016 IBM
 # Author: Harsha Thyagaraja <harshkid@linux.vnet.ibm.com>
+# Copyright: 2017 IBM
+# Authors: Narasimhan V <sim@linux.vnet.ibm.com>
```

Comment:
I wouldn't call this a big change, would you? (the purpose of authorship is to mark significant changes )